lib/scala/kg/src/main/scala/stores/kg/Neo4jKgStore.scala

```diff
@@ -102,23 +106,20 @@ final class Neo4jKgStore @Inject()(configuration: Neo4jStoreConfiguration) exten
     def toNode: KgNode = {
       val recordMap = record.asMap().asScala.toMap.asInstanceOf[Map[String, String]]
       KgNode(
-        aliases = Option(recordMap("node.aliases")).map(aliases => aliases.split(' ').toList),
-        datasource = recordMap("node.datasource"),
-        datasources = recordMap("node.datasources").split(" ").toList,
         id = recordMap("node.id"),
-        label = recordMap("node.label"),
-        other = Option(recordMap("node.other")),
-        pos = Option(recordMap("node.pos"))
+        labels = recordMap("labels").split(ListDelim).toList,
+        pos = Option(recordMap("node.pos")),
+        sources = recordMap("sources").split(ListDelim).toList
       )
     }
 
     def toPathRecord: PathRecord = {
       PathRecord(
-        datasource = record.get("path.datasource").asString(),
         objectNodeId = record.get("objectNode.id").asString(),
         pathId = record.get("path.id").asString(),
         pathEdgeIndex = record.get("path.pathEdgeIndex").asInt(),
-        pathEdgePredicate = record.get("path.pathEdgePredicate").asString(),
+        pathEdgeRelation = record.get("path.pathEdgeRelation").asString(),
+        sources = record.get("path.sources").asString().split(ListDelim).toList,
```

Comment:
neo4j uses Lucene under the hood, but I'm not sure how that interacts with property contents. Please investigate. Chances are if neo4j doesn't know about our delimiter then its Lucene won't do the tokenizing for full text search correctly.